client/components/date-range/style.scss

```diff
@@ -110,19 +110,25 @@ $date-range-mobile-layout-switch: $break-small;
 	justify-content: flex-end;
 	padding: 10px 0 0;
 
-	.button {
+	.components-button {
 		margin-left: 0.5em;
 		margin-right: 0;
```

Comment:
The margin-right seems arbitrary here and could be removed.

Reply:
Addressed in de5ffed3613